ygen/codegen.go

```diff
@@ -128,6 +128,10 @@ type ProtoOpts struct {
 	// yext.proto should be used to annotate schema paths into the output
 	// protobuf file.
 	AnnotateSchemaPaths bool
+	// AnnotateEnumNames specifies whether the extensions defined in
+	// yext.proto should be used to annotate enum values with their
+	// original YANG names in the output protobuf file.
```

Comment:
It would be good to have an explanation for the motivation behind this (and AnnotateSchemaPaths) somewhere in the code or reference a design doc.

Reply:
Added sections in https://github.com/openconfig/ygot/blob/master/docs/yang-to-protobuf-transformations-spec.md for enum values, and referenced this here.